pkg/kubelet/status_manager.go

```diff
@@ -58,6 +60,8 @@ func (s *statusManager) Start() {
 		err := s.syncBatch()
 		if err != nil {
 			glog.Warningf("Failed to updated pod status: %v", err)
+			// Errors and tight-looping are bad, m-kay
+			time.Sleep(30 * time.Second)
```

Comment:
Could this cause cluster creation to slow down significantly? I wonder if this should include increasing backoff rather than a long-ish sleep.